beeline/middleware/django/__init__.py

```diff
@@ -62,20 +62,40 @@ def create_http_event(self, request):
         trace_id, parent_id, context = _get_trace_context(request)
 
         trace_name = "django_http_%s" % request.method.lower()
+
+        meta = request.get('META')
+        remote_addr = None
+        content_length = None
+        user_agent = None
+        if meta:
+            remote_addr = meta.get('REMOTE_ADDR')
+            content_length = meta.get('CONTENT_LENGTH')
+            user_agent = meta.get('HTTP_USER_AGENT')
+
+        post = request.get('POST')
+        post_request = None
+        if post:
+            post_request = post.dict()
+
+        get = requst.get('GET')
```

Comment:
looks like this line is breaking the test

Reply:
oh there's a typo, thanks